### PR TITLE
feat(j-s): Add prosecutors, judges and registrar's emails to info cards

### PR DESCRIPTION
**Pull request by oddsson**
# Add prosecutors, judges and registrar's emails to info cards

[Asana](https://app.asana.com/0/1199153462262248/1206292245391388/f)

## What

Add prosecutors, judges and registrar's emails to info cards

## Why

If prosecutors, f.x. want to communicate with judges in a specific case, there is no way for them to contact them... Until now :) 

## Screenshots / Gifs

Screenshot 2024-01-12 at 10 59 01

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/judicial-system/web/src/components/FormProvider/limitedAccessCase.graphql [hunk 1]
```diff
@@ -54,6 +54,7 @@ query LimitedAccessCase($input: CaseQueryInput!) {
       id
       name
       title
+      email
```
Comment (unakb) [inline apps/judicial-system/web/src/components/FormProvider/limitedAccessCase.graphql:57]: Isn't the defender also allowed to see the email addresses of the judge and registrar? If so please add that to the field selectors below

File: apps/judicial-system/web/src/components/InfoCard/InfoCard.tsx [hunk 2]
```diff
@@ -47,15 +59,18 @@ const UniqueDefenders: React.FC<
       </Text>
       {uniqueDefenders.map((defender, index) =>
         defender?.name ? (
-          <Box display="flex" key={defender.name}>
-            <Text>
-              {`${defender.name}${defender.email ? `, ${defender.email}` : ''}${
-                defender.phoneNumber ? `, s. ${defender.phoneNumber}` : ''
-              }`}
+          <Box display="flex" key={defender.name} role="paragraph">
```
Comment (unakb) [inline apps/judicial-system/web/src/components/InfoCard/InfoCard.tsx:62]: Should we hide this if the user is a defender so we're not displaying his own email to him for no reason?

File: apps/judicial-system/web/src/components/FormProvider/limitedAccessCase.graphql (hunk 1 above)
Comment (oddsson) [reply apps/judicial-system/web/src/components/FormProvider/limitedAccessCase.graphql:57]: Jess! Thanks 🙏

File: apps/judicial-system/web/src/components/InfoCard/InfoCard.tsx (hunk 2 above)
Comment (oddsson) [reply apps/judicial-system/web/src/components/InfoCard/InfoCard.tsx:62]: I think we should leave it as is. You could make the case that it's good for defenders to see that the correct email and phone number is set in RVG..

**Review by unakb — APPROVED** [review]
Nice :)

**Pull request merged**